.travis.yml

```diff
@@ -6,8 +6,8 @@ python:
   - "3.5"
   - "3.6"
 
-install: 
-  - pip install . docutils pytest mock codecov==1.5.1 behave pexpect==3.3
+install:
+  - pip install . docutils pytest mock codecov==1.5.1 behave pexpect==3.3 retrying>=1.3.3
   - pip install git+https://github.com/hayd/pep8radius.git
```

Comment:
I think you could probably replace both of these pip install lines with these two so we don't have to keep them up-to-date. Just add `docutils` and `codecov` to the requirements file.
```
pip install .
pip install -r requirements-dev.txt
```

Reply:
Good point.